### PR TITLE
Fix python micro version to avoid unexpected update

### PR DESCRIPTION
**Pull request by ly-kolena**
### Linked issue(s):

I am suspecting recent (and past) sudden ci failure from broken virtualenv
> The virtual environment found in /home/circleci/project/.poetry/virtualenvs/kolena-3aSsmiER-py3.11 seems to be broken.

could be caused by docker image update that somehow creates incompatibility with the cached virtual env ([anecdotal report](https://discuss.circleci.com/t/upgrading-to-the-convenience-image-breaks-everything/42746)) 
 
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: .circleci/continue_config.yml [hunk 1]
```diff
@@ -26,20 +26,33 @@ parameters:
...
-      extras:
+        default: "3.9.18"
+      resource-class:
         type: string
-        default: "none"
+        default: "small"
     docker:
       - image: cimg/python:<< parameters.python-version >>
-    resource_class: small
+    resource_class: << parameters.resource-class >>
```
Comment (munkyshi) [inline .circleci/continue_config.yml:40]: Is there a reason why this got changed as well?

**Review by munkyshi — APPROVED** [review]
What's going to be our process for keeping these versions up to date?

**Comment by ly-kolena** [comment]
> What's going to be our process for keeping these versions up to date?

I don't expect we would have to keep an eye on micro version for our purpose here, unless there are security issues reported. That said, would have to rely on engineering team for manual tracking for now.

File: .circleci/continue_config.yml (hunk 1 above)
Comment (ly-kolena) [reply .circleci/continue_config.yml:40]: Thought to do some small refactor since this setup is repeated.

**Pull request merged**